<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>729</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="HL_topbt">
      <item>
       <widget class="QPushButton" name="pb_help">
        <property name="text">
         <string>Aide</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/plugins/Geonature-GIS-main/icons/help.png</normaloff>:/plugins/Geonature-GIS-main/icons/help.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_reset">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Réinitialiser</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/plugins/Geonature-GIS-main/icons/refresh.png</normaloff>:/plugins/Geonature-GIS-main/icons/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>525</width>
         <height>636</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="VL_viewselect">
          <item>
           <widget class="QLabel" name="lbl_selectview">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(5, 144, 110);</string>
            </property>
            <property name="text">
             <string>Sélection de l'Export * :</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="pb_selectview">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(5, 144, 110);

</string>
            </property>
            <property name="text">
             <string>Sélectionner un Export</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/Geonature-GIS-main/icons/select_export.png</normaloff>:/plugins/Geonature-GIS-main/icons/select_export.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_viewnametitle">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Nom de l'Export sélectionné :</string>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_viewname">
            <property name="text">
             <string/>
            </property>
            <property name="indent">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_viewdescriptiontitle">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Description :</string>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_viewdescription">
            <property name="text">
             <string/>
            </property>
            <property name="indent">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lbl_addfilter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(5, 144, 110);</string>
          </property>
          <property name="text">
           <string> Ajout des filtres [optionnel] :</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="pb_addfilter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(5, 144, 110);
</string>
            </property>
            <property name="text">
             <string> Ajouter des filtres</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/Geonature-GIS-main/icons/plus.png</normaloff>:/plugins/Geonature-GIS-main/icons/plus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lbl_filterparam">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>**Afficher les paramètres du filtre s'il y a lieu**</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="pb_runquery">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>  EXÉCUTER</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/Geonature-GIS-main/icons/executer_traitement.png</normaloff>:/plugins/Geonature-GIS-main/icons/executer_traitement.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lbl_typegeom">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(5, 144, 110);</string>
          </property>
          <property name="text">
           <string>Types de géométries trouvées : </string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lw_typegeomresult">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="pb_loadlayer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(5, 144, 110);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string> Charger la couche dans QGIS</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/Geonature-GIS-main/icons/view.png</normaloff>:/plugins/Geonature-GIS-main/icons/view.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_bt">
          <property name="windowModality">
           <enum>Qt::NonModal</enum>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>20</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>500</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 127);</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>7</number>
          </property>
          <property name="midLineWidth">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="GL_export">
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="2" column="2">
           <widget class="QPushButton" name="pb_export">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>EXPORTER</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/Geonature-GIS-main/icons/export_w.png</normaloff>:/plugins/Geonature-GIS-main/icons/export_w.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cb_expformat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>GeoPackage</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GeoJSON</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CSV</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>XLSX</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_format">
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_folder">
            <property name="text">
             <string>Dossier</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsFileWidget" name="qfw_folder">
            <property name="storageMode">
             <enum>QgsFileWidget::GetDirectory</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pb_quit">
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
